jetty-slf4j-impl/src/main/java/org/eclipse/jetty/logging/StdErrAppender.java

```diff
@@ -0,0 +1,331 @@
+    {
+        Throwable cause = throwable;
+
+        // Timestamp
+        ZonedDateTime tsInstant = Instant.ofEpochMilli(timestamp).atZone(timezone);
+        timestampFormatter.formatTo(tsInstant, builder);
+
+        // Level
+        builder.append(':').append(renderedLevel(level));
+
+        // Logger Name
+        builder.append(':');
+        if (condensedNames)
+        {
+            builder.append(logger.getCondensedName());
+        }
+        else
+        {
+            builder.append(logger.getName());
+        }
+
+        // Thread Name
+        builder.append(':');
+        builder.append(threadName); // TODO: support TAG_PAD configuration
+        builder.append(':');
+
+        // Message
+        builder.append(' ');
+
+        if (strictFormat)
+        {
+            FormattingTuple ft = MessageFormatter.arrayFormat(message, argumentArray);
+            appendEscaped(builder, ft.getMessage());
+            if (cause == null)
+            {
+                cause = ft.getThrowable();
+            }
+        }
+        else
+        {
+            // TODO: this should really be removed, as it violates the slf4j API contract for throwables and such
+            StringBuilder msg = new StringBuilder();
+            Object[] args = argumentArray == null ? EMPTY_ARGS : argumentArray;
+            msg.append(Objects.requireNonNullElseGet(message, () -> "{} ".repeat(args.length)));
+            String braces = "{}";
+            int start = 0;
+            for (Object arg : args)
+            {
+                int bracesIndex = msg.indexOf(braces, start);
+                if (bracesIndex < 0)
+                {
+                    appendEscaped(builder, msg.substring(start));
+                    builder.append(" ");
+                    if (arg != null)
+                        builder.append(arg);
+                    start = msg.length();
+                }
+                else
+                {
+                    appendEscaped(builder, msg.substring(start, bracesIndex));
+                    builder.append(arg);
+                    start = bracesIndex + braces.length();
+                }
+            }
+            appendEscaped(builder, msg.substring(start));
+        }
+
+        // Throwable
+        if (cause != null)
+        {
+            if (logger.isHideStacks())
+            {
+                builder.append(": ").append(cause);
+            }
+            else
+            {
+                appendCause(builder, cause, "");
+            }
+        }
+    }
+
+    private String renderedLevel(Level level)
+    {
+        switch (level)
+        {
+            case ERROR:  // New for Jetty 10+
+                return "ERROR";
+            case WARN:
+                return "WARN";
+            case INFO:
+                return "INFO";
+            case DEBUG:
+                return "DBUG"; // keeping abbreviated name for historical reasons
```

Comment:
I'm really curious about the story behind this? :) ERROR is 5 characters so DEBUG will not be longer

Reply:
ERROR is a new level that was not there before. See my comment above.

Reply:
my question is more why keep the "DBUG" :) we can use "DEBUG"

Reply:
Sure we can use "DEBUG", but we then have to use "WARN " instead of "WARN" to keep the formatting nice.  I'm OK with that... or even if we pad with "%5s" when printing (although adding the space here is probably a lot faster/simpler).

Reply:
Fixed.